Makefile

```diff
@@ -281,6 +281,12 @@ uwsgi: collectstatic
 	fi; \
     uwsgi -b 32768 --socket 127.0.0.1:8050 --module=awx.wsgi:application --home=/venv/awx --chdir=/awx_devel/ --vacuum --processes=5 --harakiri=120 --master --no-orphans --py-autoreload 1 --max-requests=1000 --stats /tmp/stats.socket --lazy-apps --logformat "%(addr) %(method) %(uri) - %(proto) %(status)" --hook-accepting1-once="exec:awx-manage run_dispatcher --reload"
 
+coverage-uwsgi: collectstatic
+	@if [ "$(VENV_BASE)" ]; then \
+		. $(VENV_BASE)/awx/bin/activate; \
+	fi; \
+    COVERAGE_ENABLED=True uwsgi -b 32768 --socket 127.0.0.1:8050 --module=awx.coverage_wsgi:application --home=/venv/awx --chdir=/awx_devel/ --vacuum --processes=5 --harakiri=120 --master --no-orphans --py-autoreload 1 --max-requests=1000 --stats /tmp/stats.socket --lazy-apps --logformat "%(addr) %(method) %(uri) - %(proto) %(status)" --hook-accepting1-once="exec:awx-manage run_dispatcher --reload"
```

Comment:
Is there any way that you could do 

```
COVERAGE_ENABLED=True make uwsgi
```

?

Ultimately, I want to continue developing this:

https://github.com/ansible/awx/compare/devel...AlanCoding:uwsgi_config

so that we will forever stop copying the parameters blob, but that still has a way to go, thus the need to hack around for the moment.

Reply:
yeah was working on that but had some hiccups, can try again

Reply:
got this fixed